tests/Unit/BaseGeoJsonTest.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace GeoJson\Tests;
+namespace GeoJson\Tests\Unit;
```

Comment:
I'd prefer to leave these namespaces as-is and minimize the diff. There are likely mixed unit and integration tests within these files, so I see no reason to attempt any segregation.

Moving all of the files from `tests/GeoJSON/Tests` to `tests/` makes sense, though (per the PSR-4 mapping you added).